Resources/views/Collector/migrations.html.twig

```diff
@@ -13,29 +11,34 @@
             {% set status_color = new_migrations > 0 ? 'red' : status_color %}
 
             {% set icon %}
-                {{ include('@DoctrineMigrations/Collector/icon.svg') }}
+                {% if profiler_markup_version == 3 %}
+                    {{ include('@DoctrineMigrations/Collector/icon-v3.svg') }}
+                {% else %}
+                    {{ include('@DoctrineMigrations/Collector/icon.svg') }}
+                {% endif %}
+
                 <span class="sf-toolbar-value">{{ new_migrations + unavailable_migrations }}</span>
             {% endset %}
 
             {% set text %}
                 <div class="sf-toolbar-info-piece">
-                    <b>Current</b>
+                    <b>Current Migration</b>
                     <span>{{ executed_migrations > 0 ? collector.data.executed_migrations|last.version|split('\\')|last : 'n/a' }}</span>
                 </div>
                 <div class="sf-toolbar-info-piece">
-                    <b>Executed</b>
+                    <b>Executed Migrations</b>
                     <span class="sf-toolbar-status">{{ executed_migrations }}</span>
                 </div>
                 <div class="sf-toolbar-info-piece">
-                    <b>Executed Unavailable</b>
+                    <b>└─ Unavailable</b>
                     <span class="sf-toolbar-status {{ unavailable_migrations > 0 ? 'sf-toolbar-status-yellow' }}">{{ unavailable_migrations }}</span>
                 </div>
                 <div class="sf-toolbar-info-piece">
-                    <b>Available</b>
+                    <b>└─ Available</b>
                     <span class="sf-toolbar-status">{{ available_migrations }}</span>
                 </div>
                 <div class="sf-toolbar-info-piece">
-                    <b>New</b>
+                    <b>└─ New</b>
```

Comment:
New migrations are not a subset of the executed migrations (the available migrations are not either btw, unless there are no new ones)